staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3788,9 +3836,17 @@ type PodSecurityContext struct {
 	// no additional groups are added to any container. Note that group memberships
 	// defined in the container image for the uid of the container process are still effective,
 	// even if they are not included in this list.
+	// If SupplementalGroupsPolicy feature gate was enabled, you can control how the groups are calculated.
 	// Note that this field cannot be set when spec.os.name is windows.
 	// +optional
 	SupplementalGroups []int64 `json:"supplementalGroups,omitempty" protobuf:"varint,4,rep,name=supplementalGroups"`
+	// Defines how supplemental groups of the first container processes are calculated.
+	// Valid values are "Merge" and "Strict". If not specified, "Merge" is used.
+	// (Alpha) Using the field requires the SupplementalGroupsPolicy feature gate to be enabled
+	// and the container runtime must implement support for this feature.
+	// +featureGate=SupplementalGroupsPolicy
+	// +optional
```

Comment:
I think you can `+default=Merge` here and avoid manual defaulting

Reply:
I think `SupplementalGroupsPolicy` is os dependent. We should reject this field when os is windows. So, I didn't put the `+default` tag and keeps windows validation.

https://github.com/kubernetes/kubernetes/pull/117842/files#diff-c713e8919642d873fdf48fe8fb6d43e5cb2f53fd601066ff53580ea655948f0dR4283-R4285